mat64/cholesky.go

```diff
@@ -328,11 +371,15 @@ func (c *Cholesky) SymRankOne(orig *Cholesky, alpha float64, x *Vector) (ok bool
 	if ok {
 		c.updateCond(-1)
 	} else {
-		c.cond = math.Inf(1)
+		c.Reset()
 	}
 	return ok
 }
 
 func (c *Cholesky) isZero() bool {
 	return c.chol == nil
 }
+
+func (c *Cholesky) valid() bool {
```

Comment:
Can you check if this gets inlined? If it doesn't make a checker method that includes the panic to save some lines in the preambles above.